exercises/practice/dot-dsl/src/main/java/DotDsl.java

```diff
@@ -0,0 +1,65 @@
+import java.util.Collection;
+import java.util.Map;
+
+class DotDsl {
+
+    static class Graph {
+        public Graph() {
+        }
+
+        public Graph(Map<String, String> attributes) {
+        }
+
+        public Collection<Node> getNodes() {
+            throw new UnsupportedOperationException();
```

Comment:
It will be great to add the following message `Delete this statement and write your own implementation.` to all the `UnsupportedOperationException`
```suggestion
            throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
```